www/nginx/pkg-descr

```diff
@@ -10,6 +10,10 @@ WWW: https://nginx.org/
 Plugin Changelog
 ================
 
+1.24.1
```

Comment:
you can leave this at 1.24... it's the new version anyway :)

Reply:
hi! oops. already bumped the revision. should i revert to the main ver?

Reply:
since 1.24 is just one hour old and no release yet it's better to move everything together into 1.24 indeed

Reply:
got it. i'll revert it now, thanks)